ui/src/app/components/ct-available-banner-maybe.tsx

```diff
@@ -34,9 +39,7 @@ const shouldShowBanner = (
     profile &&
     ct &&
     // the environment allows users to see the CT (in the UI)
```

Comment:
Should this comment be updated now that config is coming from API?

Reply:
I think it's still true in the general sense: not `environment.ts` but it's the config settings for the particular environment.  LMK if you think I can say this in a better way.

Reply:
ok so the phrase 'in the UI' means where the user will be able to see the CT, not where the environment configuration is coming from?

Reply:
oh I see what you mean - we no longer restrict to "in the UI" so I'll remove this.